src/main/java/org/springframework/data/repository/util/ReactiveWrapperConverters.java

```diff
@@ -722,7 +722,7 @@ public Completable convert(Publisher<?> source) {
 
 		@Override
 		public io.reactivex.Observable<?> convert(Publisher<?> source) {
-			return (io.reactivex.Observable<?>) REACTIVE_ADAPTER_REGISTRY.getAdapter(io.reactivex.Single.class)
+			return (io.reactivex.Observable<?>) REACTIVE_ADAPTER_REGISTRY.getAdapter(io.reactivex.Observable.class)
```

Comment:
That's the converter to adopt a `Publisher` to an `Observable`. `Flowable` adoption happens in the [`PublisherToRxJava2FlowableConverter `](https://github.com/spring-projects/spring-data-commons/blob/2.0.x/src/main/java/org/springframework/data/repository/util/ReactiveWrapperConverters.java#L864). That's a fix for a bug we didn't noticed earlier.